website/source/docs/agent/configuration/consul.html.md

```diff
@@ -103,7 +103,9 @@ configuration, Nomad will automatically connect and configure with Consul.
   communicate with the Consul agent.
 
 - `token` `(string: "")` - Specifies the token used to provide a per-request ACL
-  token. This option overrides the Consul Agent's default token.
+  token. This option overrides the Consul Agent's default token (beware that the 
```

Comment:
What about:

```
This option overrides the Consul Agent's default token. If the token is not set here or on the Consul agent, it will default to Consul's anonymous policy, which may or may not allow writes.
```

Reply:
Even better!